src/main/webapp/WEB-INF/jsp/login.jsp

```diff
@@ -0,0 +1,35 @@
+<% boolean error = (boolean) request.getAttribute("error"); %>
```

Comment:
This and other files: Consider pulling CSS out into a separate .css file. Up to you, though. This is fine for now, but will be harder to scale.